tools/gentest.ml

```diff
@@ -398,8 +398,7 @@ let () =
      ; "--no-minimal-bj" ])
   ; ("runtest", "optimize",
      [ "--output=smtlib2"
-     ; "--frontend dolmen"
-     ; "--optimize" ])]
+     ; "--frontend dolmen" ])]
```

Comment:
We should remove the `optimize` tag in our test suite.

Reply:
Done.